selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/core/provider/CheTestIsMultiuserProvider.java

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright (c) 2012-2017 Red Hat, Inc.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *   Red Hat, Inc. - initial API and implementation
+ */
+package org.eclipse.che.selenium.core.provider;
+
+import com.google.inject.Inject;
+import com.google.inject.Provider;
+import javax.inject.Named;
+
+/** @author Dmytro Nochevnov */
+public class CheTestIsMultiuserProvider implements Provider<Boolean> {
```

Comment:
What advantages does have this provider ?
Why not to inject `che.multiuser` property wherever it needed?

Reply:
`@Named("che.multiuser")` is error-prone command so as because of nothing prevents from typo in  variable name in case of multiple usage.

Reply:
Could you elaborate on more?

Reply:
It's kind of anti-pattern _Magic String_.